libs/iplant-clojure-commons/project.clj

```diff
@@ -6,6 +6,9 @@
   :dependencies [[org.clojure/clojure "1.7.0"]
                  [org.clojure/tools.logging "0.3.1"]
                  [buddy/buddy-sign "0.7.0"]
+                 [org.clojure/data.json "0.2.6"]
```

Comment:
I could be missing something, but it doesn't look as though the `org.clojure/data.json` dependency is used anywhere in `iplant-clojure-commons`. We've pretty much settled on `cheshire` instead.